### PR TITLE
Add Ryot plugin

### PR DESCRIPTION
**Pull request by bretthysuik**
A plugin for [Flow Launcher](https://github.com/Flow-Launcher/Flow.Launcher) to search your [ryot](https://github.com/IgnisDa/ryot) tracker.

image

### CODE REVIEW

File: plugins.json [hunk 1]
```diff
@@ -1704,5 +1704,18 @@
...
+        "Name": "ryot",
+        "Description": "Search your ryot tracker",
+        "Author": "bretthysuik",
+        "Version": "1.0.0",
+        "Language": "csharp",
+        "Website": "https://github.com/bretthysuik/Flow.Launcher.Plugin.Ryot",
+        "UrlDownload": "https://github.com/bretthysuik/Flow.Launcher.Plugin.Ryot/releases/download/v1.0.0/Flow.Launcher.Plugin.Ryot.zip",
+        "UrlSourceCode": "https://github.com/bretthysuik/Flow.Launcher.Plugin.Ryot/tree/main",
+        "IcoPath": "https://cdn.jsdelivr.net/gh/bretthysuik/Flow.Launcher.Plugin.Ryot@main/Flow.Launcher.Plugin.Ryot/Images/ryot.png",
+        "DateAdded": "2023-09-04T20:39:20"
```
Comment (jjw24) [inline plugins.json:1719]:
```suggestion
```

File: plugins.json [hunk 2]
```diff
@@ -1704,5 +1704,17 @@
...
+        "ID": "BC4E22D7C0CD4FBA9108C26B8B8DBA8D",
+        "Name": "ryot",
+        "Description": "Search your ryot tracker",
+        "Author": "bretthysuik",
+        "Version": "1.0.0",
+        "Language": "csharp",
+        "Website": "https://github.com/bretthysuik/Flow.Launcher.Plugin.Ryot",
+        "UrlDownload": "https://github.com/bretthysuik/Flow.Launcher.Plugin.Ryot/releases/download/v1.0.0/Flow.Launcher.Plugin.Ryot.zip",
+        "UrlSourceCode": "https://github.com/bretthysuik/Flow.Launcher.Plugin.Ryot/tree/main",
+        "IcoPath": "https://cdn.jsdelivr.net/gh/bretthysuik/Flow.Launcher.Plugin.Ryot@main/Flow.Launcher.Plugin.Ryot/Images/ryot.png",
```
Comment (jjw24) [inline plugins.json:1718]:
```suggestion
        "IcoPath": "https://cdn.jsdelivr.net/gh/bretthysuik/Flow.Launcher.Plugin.Ryot@main/Flow.Launcher.Plugin.Ryot/Images/ryot.png"
```

**Comment by jjw24** [comment]
Hi there, the plugin fails to load after installing. It failed because the InitAsync method loads the url and token into `new GraphQL.Client.Http.GraphQLHttpClient`, which then complains due to the two settings are blank strings- format of URI.

Should be able to repro this by removing your plugin's existing settings file so it loads fresh.


`20:17:22.5337+10:00 - ERROR - PluginManager.InitializePluginsAsync - Fail to Init plugin: ryot
EXCEPTION OCCURS: System.UriFormatException: Invalid URI: The format of the URI could not be determined.
   at void Uri.CreateThis(string uri, bool dontEscape, UriKind uriKind, in UriCreationOptions creationOptions)
   at new Uri(string uriString)
   at new GraphQL.Client.Http.GraphQLHttpClient(string endPoint, IGraphQLWebsocketJsonSerializer serializer) in /_/src/GraphQL.Client/GraphQLHttpClient.cs:line 48
   at new Flow.Launcher.Plugin.Ryot.Ryot(Settings settings)
   at Task Flow.Launcher.Plugin.Ryot.RyotPlugin.InitAsync(PluginInitContext context)
   at Task Flow.Launcher.Core.Plugin.PluginManager.InitializePluginsAsync(IPublicAPI api)+() => { } in C:/projects/flow-launcher/Flow.Launcher.Core/Plugin/PluginManager.cs:line 126
   at async Task<long> Flow.Launcher.Infrastructure.Stopwatch.DebugAsync(string message, Func<Task> action) in C:/projects/flow-launcher/Flow.Launcher.Infrastructure/Stopwatch.cs:line 34
   at async Task Flow.Launcher.Core.Plugin.PluginManager.InitializePluginsAsync(IPublicAPI api)+(?) => { } in C:/projects/flow-launcher/Flow.Launcher.Core/Plugin/PluginManager.cs:line 125`

Tip: 
A good idea to handle this scenario is to validate the settings loaded first, if the token and baseURL not added then show a result informing the user to put these two in the plugin settings.

**Comment by bretthysuik** [comment]
OK thanks, I'll make that change.

**Comment by bretthysuik** [comment]
@jjw24 I've fixed the problem you identified

**Review by jjw24 — APPROVED** [review]
👍 thanks for making the change

**Pull request merged**